data/faq.json

```
@@ -2,49 +2,49 @@
   {
     "id": 1,
     "question": "What is JSON Schema?",
-    "answer": "JSON Schema is a vocabulary that allows you to annotate and validate JSON documents. It defines the structure, data types, and constraints of JSON data.",
+    "answer": "JSON Schema is a specification for defining the structure of JSON data. It provides a standardized way to annotate, describe, and validate JSON data. JSON Schema defines content, structure, data types, and other expected constraints within a JSON document. The key aspects of JSON Schema include defining data structure, validation documentation data contracts, and diverse use cases.",
     "category": "general"
   },
   {
     "id": 2,
     "question": "How do I create a simple JSON Schema?",
-    "answer": "You can create a simple JSON Schema using a JSON object with properties like 'type', 'properties', and 'required'. These define the data type, properties, and mandatory fields of your JSON document.",
+    "answer": "You create a simple JSON Schema by defining keywords such as 'schema', 'id', 'title', 'description', and 'type', which specify the version, unique identifier, title, and, description of your schema. Then, specify the data type for your schema will validate (e.g., object, array) and define the necessary properties, including their types and constraints. You can also include keywords like; 'properties', 'required', and 'default'. These define the constraints of your JSON document.",
```

```suggestion
    "answer": "You can create a simple JSON Schema using a JSON object with properties like 'type', 'properties', and 'required'. These define the data type, properties, and mandatory fields of your JSON document.",
```

Let's keep this one simple.